### PR TITLE
Added Simulation Features

### PR DESCRIPTION
**Pull request by Lobotuerk**
Signed-off-by: Lobotuerk <jtlorente@ekumenlabs.com>

### CODE REVIEW

File: bullet/src/SimulationFeatures.cc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2019 Open Source Robotics Foundation
```
Comment (Blast545) [inline bullet/src/SimulationFeatures.cc:2]:
```suggestion
 * Copyright (C) 2020 Open Source Robotics Foundation
```

File: bullet/src/SimulationFeatures.hh [hunk 2]
```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2019 Open Source Robotics Foundation
```
Comment (Blast545) [inline bullet/src/SimulationFeatures.hh:2]:
```suggestion
 * Copyright (C) 2020 Open Source Robotics Foundation
```

**Review by Blast545 — APPROVED** [review]
LGTM

**Comment by Lobotuerk** [comment]
Merging

**Pull request merged**